runtime/jcl/common/jclvm.c

```diff
@@ -51,12 +51,24 @@ typedef struct AllInstancesData {
 	UDATA instanceCount;        /* count of instances found regardless if they are being collected in the array or not */
 } AllInstancesData;
 
+typedef struct J9HeapStatisticsTableEntry {
+	J9Class *klass; /* hash table key */
```

Comment:
Pleas use `clazz` instead of `klass` like the rest of this file.

Reply:
I also changed arrayKlass